LICENSE-3rdparty.yml

```diff
@@ -1,4 +1,4 @@
-root_name: datadog-opentelemetry, datadog-opentelemetry-mappings, dd-trace, dd-trace-propagation, dd-trace-sampling
+root_name: datadog-opentelemetry, datadog-opentelemetry-mappings, dd-trace, dd-trace-propagation, dd-trace-sampling, propagator
```

Comment:
I don't understand how the license tool is working, but not including this causes a diff 🤷

Reply:
Yeah we should probably remove the `.yml` one and only have the `.csv` one.